client/src/components/AuthHeader/AuthHeader.tsx

```diff
@@ -43,7 +44,9 @@ const AuthHeader = ({ linkTo, btnText }: Props): JSX.Element => {
             </Typography>
           </Link>
           <Typography className={classes.navLink} color="secondary" display="inline">
-            <NotificationPopUp />
+            <NotificationProvider>
```

Comment:
Why are we wrapping this component with the Provider as we've already wrapped the Provider around the entire app?  :)

Reply:
removed the wrapping in the last commit